packages/block-library/src/navigation/constants.js

```diff
@@ -37,3 +37,5 @@ export const SELECT_NAVIGATION_MENUS_ARGS = [
 	'wp_navigation',
 	PRELOADED_NAVIGATION_MENUS_QUERY,
 ];
+
+export const NAVIGATION_MOBILE_BREAKPOINT = 600;
```

Comment:
Should we try and establish a consistent terminology here of "collapsed" as you have elsewhere?